camel/utils/commons.py

```diff
@@ -241,3 +242,93 @@ def check_server_running(server_url: str) -> bool:
 
     # if the port is open, the result should be 0.
     return result == 0
+
+
+def dependencies_required(*required_modules: str) -> Callable[[F], F]:
+    r"""A decorator to ensure that specified Python modules
+    are available before a function executes.
+
+    Parameters:
+    required_modules (str): The required modules to be checked for
+    availability.
+
+    Returns:
+    Callable[[F], F]: The original function with the added check for
+    required module dependencies.
+
+    Raises:
+    ImportError: If any of the required modules are not available.
+
+    Example:
+    @dependencies_required('numpy', 'pandas')
+    def data_processing_function():
+        # Function implementation...
```

Comment:
Our project usually uses google style docstring. Could you please fix them?

Here is an example:

```suggestion
    Args:
        required_modules (str): The required modules to be checked for
            availability.

    Returns:
        Callable[[F], F]: The original function with the added check for
            required module dependencies.

    Raises:
        ImportError: If any of the required modules are not available.

    Example:
        ::
        
            @dependencies_required('numpy', 'pandas')
            def data_processing_function():
                # Function implementation...
```